### ISSUE TITLE
laplace_transform is dropping a constant in some cases

### ISSUE THREAD

**Issue by atulprak** [sympy/sympy#24415]
The following code does not work correctly. The laplace_transform function is ignoring the constant 1000. Changing it does not change the Laplace transform either.

```
import sympy
from sympy.abc import s, t

f  = 1000*sympy.exp(-t)*sympy.Heaviside(t)
fg = sympy.laplace_transform(f, t, s)
print("f = ", f)
print("laplace of f = ", fg)

print("Incorrect laplace. The constant 1000 is missing in the answer.");
```
<img width="450" alt="Output" src="https://user-images.githubusercontent.com/2295189/209046227-f61d1d9e-0170-4ea4-bd4b-d5c5d1ce0b23.png">

**Comment by 1e9abhi1e10** [comment]
Could you please check once again because in my system it gives
```
f =  1000*exp(-t)*Heaviside(t)
laplace of f =  (1000/(s + 1), 0, True)
Incorrect laplace. The constant 1000 is missing in the answer.

**Comment by oscarbenjamin** [comment]
This is fixed on master since SymPy 1.11.1 was released. The fix will be in SymPy 1.12.

Thanks for reporting!
